### ISSUE TITLE
Adding files / images does not work

### ISSUE THREAD

**Issue by macbih** [hbrennhaeuser/homeassistant_integration_ntfy#12]
First of all thank you for your awesome integration!

My setup basically does work. The integration is able to send messages including e.g. setting priority. But I am unfortunately not able to attach images to it!

While this notification with Pushover works flawlessly (I would like to replace it with ntfy)

```
      - action: notify.pushover
        data:
          title: Entrance
          message: Person detected!
          data:
            attachment: "/media/kameras/doorbell/person.jpg"
```

this one (which is almost identical) fails and just sends title and message:

```
      - action: notify.ntfy
        data:
          title: Entrance
          message: Person detected!
          data:
            attach_file: "/media/kameras/doorbell/person.jpg"
            attachment_filename: 'person.jpg'
```

According to the docs the syntax should be okay but it fails. I tried to add `/local` to the image path but saw no difference. 

I am also able to send notifications including images or files from the [ntfy.sh account page](https://ntfy.sh/account) onto my topic.

My account type is Free. This is my setup (using https://ntfy.sh):

```
- name: ntfy
  platform: ntfy
  url: !secret ntfy_url
  authentication: 'token'
  token: !secret ntfy_token
  topic: mytopic
  #verify_ssl: True 
  allow_topic_override: True
  #attachment_maxsize: 2M
```

What am I doing wrong? I apologize if I have overlooked something trivial. Can you assist, please? Thank you!

**Comment by hbrennhaeuser** [comment]
What version of the integration are you using?

Attachments are only supported in v1.1.0, which is currently only available as a pre-release. If you are using HACS you probably have to manually select the pre-release-version (Redownload -> Need a different version?).

This could easily be missed as the main branch readme already shows the options for v1.1.0 but HACS downloads the latest stable (v1.0.2) by default. 

If you are on v1.1.0 already, can you check if there are any error-messages in the Home Assistant logs?

**Comment by macbih** [comment]
I appreciate your immediate answer! Thank you!

> What version of the integration are you using?

In fact, it was `1.0.2`. It is true that I did not realise I needed current pre-release [v1.1.0-pre.2](https://github.com/hbrennhaeuser/homeassistant_integration_ntfy/releases/tag/v1.1.0-pre.2) for this functionality. I missed that one. Now I installed this version and it works! Many thanks again! I am happy now! 

Please note that my above mentioned code works as is. There is no `/local` in the file path.

**Comment by macbih** [comment]
My issue has been solved with the current pre-release [v1.1.0-pre.2](https://github.com/hbrennhaeuser/homeassistant_integration_ntfy/releases/tag/v1.1.0-pre.2). Thank you!

In case there are no other things in the pipeline it may make sense to publish a new release version `1.1.0` so that it also matches the documention. 😉
